app/src/main/java/io/intrepid/contest/screens/contestoverview/ContestOverviewPresenter.java

```diff
@@ -0,0 +1,60 @@
+package io.intrepid.contest.screens.contestoverview;
+
+import android.support.annotation.NonNull;
+
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BasePresenter;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.reactivex.disposables.Disposable;
+import timber.log.Timber;
+
+public class ContestOverviewPresenter extends BasePresenter<ContestOverviewContract.View>
+        implements ContestOverviewContract.Presenter {
+
+    private String contestId;
+
+    public ContestOverviewPresenter(@NonNull ContestOverviewContract.View view,
+                                    @NonNull PresenterConfiguration configuration) {
+        super(view, configuration);
+    }
+
+    @Override
+    public void onViewCreated() {
+        super.onViewCreated();
+
+        contestId = persistentSettings.getCurrentContestId().toString();
+        requestContestDetails();
+        requestContestStatus();
+    }
+
+    private void requestContestStatus() {
```

Comment:
super nit but can you make the order of the  two "request" method definitions match the order you're calling them in.